Warning/Settlement/MissingFile.ts

```diff
@@ -0,0 +1,31 @@
+import { isoly } from "isoly"
+import { isly } from "isly"
+import { Stack } from "../../Card/Stack"
+import { Batch } from "../../Settlement/Batch"
+import { Identifier } from "../../Settlement/Identifier"
+import { Totals } from "../../Settlement/Totals"
+import { Base } from "../Base"
+
+export interface MissingFile extends Base {
+	type: "missing-file"
+	resource: Identifier
+	batch: Batch
```

Comment:
Don't have batch here, we need to remove it from entries soon anyway since  it's a mastercard specific thing